src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetFiltersMappingPage.tsx

```diff
@@ -444,17 +460,17 @@ export default function ReleaseApiDataSetFiltersMappingPage() {
 
             <h3
               className="govuk-heading-l govuk-!-margin-top-8"
-              id="auto-mapped-filter-options"
+              id={sectionIds.autoMappedFilterOptions}
             >
               Auto mapped filter options ({totalAutoMappedFilterOptions}){' '}
               <Tag colour="grey">No action required</Tag>
             </h3>
 
             {totalAutoMappedFilterOptions > 0 ? (
               <Accordion
-                id="auto-mapped-filter-options-accordion"
+                id={`${sectionIds.autoMappedFilterOptions}-accordion`}
```

Comment:
As mentioned before, worth a variable for this?  Same comments for other such changes in this PR :)